driver-core/src/main/com/mongodb/internal/connection/DefaultConnectionPool.java

```diff
@@ -194,34 +194,34 @@ public InternalConnection get(final OperationContext operationContext) {
 
     @Override
     public InternalConnection get(final OperationContext operationContext, final long timeoutValue, final TimeUnit timeUnit) {
-        TimePoint checkoutStart = connectionCheckoutStarted(operationContext);
-        Timeout timeout = Timeout.started(timeoutValue, timeUnit, checkoutStart);
+        TimePoint tConnectionCheckOutStarted = connectionCheckoutStarted(operationContext);
```

Comment:
What's up with the `t` prefix?  Is it Hungarian notation?  It's not something we generally use in this project.

Reply:
I simply used the name used in pseudocode in the spec, to facilitate matching them.

Reply:
I'd prefer that we stick to our own naming conventions.

Reply:
Done.